instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/instrumentation.rb

```diff
@@ -12,7 +12,7 @@ module GraphQL
       # The Instrumentation class contains logic to detect and install the GraphQL instrumentation
       class Instrumentation < OpenTelemetry::Instrumentation::Base
         compatible do
-          gem_version < Gem::Version.new('3.0.0')
+          gem_version <= Gem::Version.new('2.0.16')
```

Comment:
is the PR Title incorrect? if not then shouldn't it be: 
```suggestion
          gem_version <= Gem::Version.new('2.0.17')
```